[2024-10-16T16:13:04.667+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T16:13:04.697+0000] {__init__.py:51} DEBUG - Loading core task runner: StandardTaskRunner
[2024-10-16T16:13:04.718+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T16:13:04.728+0000] {base_task_runner.py:68} DEBUG - Planning to run as the  user
[2024-10-16T16:13:04.731+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [queued]> from DB
[2024-10-16T16:13:04.757+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2024-10-16T16:13:04.763+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-16T16:13:04.764+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2024-10-16T16:13:04.776+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-16T16:13:04.776+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [queued]>
[2024-10-16T16:13:04.777+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2024-10-16T16:13:04.779+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-16T16:13:04.786+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-16T16:13:04.791+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2024-10-16T16:13:04.791+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [queued]>
[2024-10-16T16:13:04.791+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-10-16T16:13:04.805+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): model.dbt_btc_transform.stg_kaggle__btcusdt> on 2024-10-16 16:13:01.431647+00:00
[2024-10-16T16:13:04.809+0000] {standard_task_runner.py:57} INFO - Started process 189 to run task
[2024-10-16T16:13:04.812+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'run_dbt_models', 'model.dbt_btc_transform.stg_kaggle__btcusdt', 'manual__2024-10-16T16:13:01.431647+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt_models.py', '--cfg-path', '/tmp/tmp87ff591h']
[2024-10-16T16:13:04.813+0000] {standard_task_runner.py:85} INFO - Job 57: Subtask model.dbt_btc_transform.stg_kaggle__btcusdt
[2024-10-16T16:13:04.814+0000] {cli_action_loggers.py:65} DEBUG - Calling callbacks: [<function default_action_log at 0x7f7ca9505550>]
[2024-10-16T16:13:04.873+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T16:13:04.882+0000] {task_command.py:410} INFO - Running <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [running]> on host 888d2c584607
[2024-10-16T16:13:04.882+0000] {settings.py:353} DEBUG - Disposing DB connection pool (PID 189)
[2024-10-16T16:13:04.883+0000] {settings.py:212} DEBUG - Setting up DB connection pool (PID 189)
[2024-10-16T16:13:04.883+0000] {settings.py:285} DEBUG - settings.prepare_engine_args(): Using NullPool
[2024-10-16T16:13:04.885+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [running]> from DB
[2024-10-16T16:13:04.932+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-16T16:13:04.940+0000] {taskinstance.py:868} DEBUG - Clearing XCom data
[2024-10-16T16:13:04.973+0000] {retries.py:80} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2024-10-16T16:13:04.989+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_models' AIRFLOW_CTX_TASK_ID='model.dbt_btc_transform.stg_kaggle__btcusdt' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T16:13:01.431647+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-16T16:13:01.431647+00:00'
[2024-10-16T16:13:04.990+0000] {__init__.py:117} DEBUG - Preparing lineage inlets and outlets
[2024-10-16T16:13:04.990+0000] {__init__.py:155} DEBUG - inlets: [], outlets: []
[2024-10-16T16:13:04.991+0000] {bash.py:186} DEBUG - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_models' AIRFLOW_CTX_TASK_ID='model.dbt_btc_transform.stg_kaggle__btcusdt' AIRFLOW_CTX_EXECUTION_DATE='2024-10-16T16:13:01.431647+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-16T16:13:01.431647+00:00'
[2024-10-16T16:13:04.991+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-10-16T16:13:04.992+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            export DBT_SCHEMA=analytics &&\n            cd /opt/airflow/dbt &&\n            /home/airflow/.local/bin/dbt --no-write-json run --target dev --models stg_kaggle__btcusdt\n            ']
[2024-10-16T16:13:05.000+0000] {subprocess.py:86} INFO - Output:
[2024-10-16T16:13:07.828+0000] {subprocess.py:93} INFO - [0m16:13:07  Running with dbt=1.8.7
[2024-10-16T16:13:08.323+0000] {subprocess.py:93} INFO - [0m16:13:08  Registered adapter: ***=1.8.2
[2024-10-16T16:13:08.497+0000] {subprocess.py:93} INFO - [0m16:13:08  Unable to do partial parsing because of a version mismatch
[2024-10-16T16:13:09.838+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [running]> from DB
[2024-10-16T16:13:09.853+0000] {job.py:213} DEBUG - [heartbeat]
[2024-10-16T16:13:10.749+0000] {subprocess.py:93} INFO - [0m16:13:10  Found 1 model, 1 source, 540 macros
[2024-10-16T16:13:10.751+0000] {subprocess.py:93} INFO - [0m16:13:10
[2024-10-16T16:13:10.979+0000] {subprocess.py:93} INFO - [0m16:13:10  Concurrency: 4 threads (target='dev')
[2024-10-16T16:13:10.980+0000] {subprocess.py:93} INFO - [0m16:13:10
[2024-10-16T16:13:10.991+0000] {subprocess.py:93} INFO - [0m16:13:10  1 of 1 START sql table model analytics.stg_kaggle__btcusdt ..................... [RUN]
[2024-10-16T16:13:13.218+0000] {subprocess.py:93} INFO - [0m16:13:13  1 of 1 OK created sql table model analytics.stg_kaggle__btcusdt ................ [[32mSELECT 999999[0m in 2.22s]
[2024-10-16T16:13:13.239+0000] {subprocess.py:93} INFO - [0m16:13:13
[2024-10-16T16:13:13.240+0000] {subprocess.py:93} INFO - [0m16:13:13  Finished running 1 table model in 0 hours 0 minutes and 2.49 seconds (2.49s).
[2024-10-16T16:13:13.242+0000] {subprocess.py:93} INFO - [0m16:13:13
[2024-10-16T16:13:13.243+0000] {subprocess.py:93} INFO - [0m16:13:13  [32mCompleted successfully[0m
[2024-10-16T16:13:13.244+0000] {subprocess.py:93} INFO - [0m16:13:13
[2024-10-16T16:13:13.246+0000] {subprocess.py:93} INFO - [0m16:13:13  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-10-16T16:13:13.934+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-16T16:13:13.965+0000] {__init__.py:75} DEBUG - Lineage called with inlets: [], outlets: []
[2024-10-16T16:13:13.966+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.model.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [running]> from DB
[2024-10-16T16:13:13.979+0000] {taskinstance.py:1356} DEBUG - Clearing next_method and next_kwargs.
[2024-10-16T16:13:13.980+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=run_dbt_models, task_id=model.dbt_btc_transform.stg_kaggle__btcusdt, execution_date=20241016T161301, start_date=20241016T161304, end_date=20241016T161313
[2024-10-16T16:13:13.980+0000] {taskinstance.py:2305} DEBUG - Task Duration set to 9.202776
[2024-10-16T16:13:13.994+0000] {cli_action_loggers.py:83} DEBUG - Calling callbacks: []
[2024-10-16T16:13:14.008+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-10-16T16:13:14.065+0000] {dagrun.py:697} DEBUG - number of tis tasks for <DagRun run_dbt_models @ 2024-10-16 16:13:01.431647+00:00: manual__2024-10-16T16:13:01.431647+00:00, state:running, queued_at: 2024-10-16 16:13:01.953177+00:00. externally triggered: True>: 4 task(s)
[2024-10-16T16:13:14.066+0000] {dagrun.py:718} DEBUG - number of scheduleable tasks for <DagRun run_dbt_models @ 2024-10-16 16:13:01.431647+00:00: manual__2024-10-16T16:13:01.431647+00:00, state:running, queued_at: 2024-10-16 16:13:01.953177+00:00. externally triggered: True>: 2 task(s)
[2024-10-16T16:13:14.068+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-16T16:13:01.431647+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-16T16:13:14.068+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-16T16:13:01.431647+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-16T16:13:14.068+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-16T16:13:01.431647+00:00 [None]> dependency 'Trigger Rule' PASSED: False, Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=1, skipped=0, failed=0, upstream_failed=0, removed=0, done=1), upstream_task_ids={'model.dbt_btc_transform.stg_kaggle__btcusdt', 'test.dbt_btc_transform.stg_kaggle__btcusdt'}
[2024-10-16T16:13:14.069+0000] {taskinstance.py:1093} DEBUG - Dependencies not met for <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-16T16:13:01.431647+00:00 [None]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_states=_UpstreamTIStates(success=1, skipped=0, failed=0, upstream_failed=0, removed=0, done=1), upstream_task_ids={'model.dbt_btc_transform.stg_kaggle__btcusdt', 'test.dbt_btc_transform.stg_kaggle__btcusdt'}
[2024-10-16T16:13:14.070+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-16T16:13:14.070+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-16T16:13:14.073+0000] {taskinstance.py:1103} DEBUG - Dependencies all met for dep_context=None ti=<TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-16T16:13:01.431647+00:00 [None]>
[2024-10-16T16:13:14.081+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
